packaging/centos-6/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-6:latest
+MAINTAINER support@osuosl.org
+RUN yum -y install \
+    automake \
+    gcc \
+    gcc-c++ \
+    glibc \
+    glibc-devel \
+    glibc-static \
+    make \
+    tar \
+    wget \
+    zlib \
+    zlib-devel \
+    zlib-static \
+    && yum clean all
+RUN mkdir /dssp
+WORKDIR /dssp
+RUN wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz && \
+    tar zxf bzip2-1.0.6.tar.gz
```

Comment:
mkdssp, the binary we're making, is statically linked.  It requires static libraries from bzip2 and boost.  The Centos packages for both of these products only include dynamic libraries, not static ones.

In the future, a pull request to convert mkdssp to dynamic linking would make the installation of bzip2 and boost unnecessary at this point but would them require the addition of dependencies on the bzip2 and boost packages.